Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.8-slim
+FROM python:3.8
```

Comment:
That usually saves us a decent amount of docker image size, is this mandatory?

Reply:
Unless we install git with apt. Not sure it will save us a lot of space. Once a new release of the API is available, we can move back to -slim